src/Controls/src/Core/IVisualTreeHelper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Microsoft.Maui.Controls
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public interface IVisualTreeHelper
+	{
+		IReadOnlyList<object> GetVisualChildren();
```

Comment:
`object` might be a bit too broad. You can use `IView`, or `IElement` if you need to include windows as "visual children".

Reply:
I agree. Since everything is based on `IElement` and that's the base, we should be safe using that instead of `object`

Reply:
IElement is internal, are you ok with the interface being internal too @drasticactions ? also, TableSection isn't an Element, but you used to return it in the other implementation...

@drasticactions please change this PR the way you prefer it, that'd be easier

Reply:
IElement is public